students/eoner/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,15 @@
+# Write a program that prints the numbers from 1 to 100 inclusive.
+# But for multiples of three print “Fizz” instead of the number.
+# For the multiples of five print “Buzz” instead of the number.
+# For numbers which are multiples of both three and five print “FizzBuzz” instead.
+
+def fizbuzz(l=100):
+    for i in range(l+1):
+        if i % 3 == 0 and i % 5 == 0:
+            print("FizzBuzz")
+        elif i % 3 == 0:
+            print("Fizz")
+        elif i % 5 == 0:
+            print("Buzz")
+        else:
+            print(i)
```

Comment:
I'd appreciate if you also included testing, but this is still fine. I like the default argument used.